addons/attach/functions/fnc_attach.sqf

```diff
@@ -26,14 +26,21 @@ if ((_itemClassname == "") || {(!_silentScripted) && {!(_this call FUNC(canAttac
 
 private _itemVehClass = getText (configFile >> "CfgWeapons" >> _itemClassname >> "ACE_Attachable");
 private _onAttachText = getText (configFile >> "CfgWeapons" >> _itemClassname >> "displayName");
+private _itemModelOrientation = getArray (configFile >> "CfgWeapons" >> _itemClassname >> "ACE_attachable_orientation");
```

Comment:
Would prefer `QGVAR(orientation)` for the property.

Reply:
I can change to use that. However that means that the orientation attribute in config would be named ``ace_attach_orientation`` which would break with the current scheme of ``ACE_Attachable`` config value.

example:
```cpp
        ACE_Attachable = "new_attachable_item_classname";
        ace_attach_orientation[] = {-1,0,0};
```

I assume you do not want to change the ``ACE_Attachable`` in line with ``QGVAR`` as that would be a breaking change. 

Let me know if its fine to have that difference and I will change to use ``QGVAR(orientation)``.

Reply:
Nowadays, new attributes usually use the component prefixes. Therefore, I think updating it to `QGVAR(orientation)` is for the best.